content/src/content/jcr_root/apps/core/email/components/segmentation/v1/segmentation/segmentation.html

```diff
@@ -58,6 +58,7 @@
              data-cmp-hook-tabs="tabpanel"
              data-cmp-data-layer="${item.data.json}"></sly>
     </table>
-    <sly data-sly-resource="${resource.path @ resourceType='wcm/foundation/components/parsys/newpar', appendPath='/*', decorationTagName='div', cssClassName='new section aem-Grid-newComponent'}"
+    <sly data-sly-resource="${resource.path @ resourceType='wcm/foundation/components/parsys/newpar', appendPath='/*', decorationTagName='div', cssClassName='new section aem-Grid-newComponent cmp-segmentation'}"
```

Comment:
Why not hiding the whole component `<div>` when the component is empty?

Reply:
The `<div>` is required for the insert component action. If it is not getting rendered the`add segments` action in the component dialog will not work.